docs/wiring/blower.rst

```diff
@@ -0,0 +1,43 @@
+.. _Blower Harness:
+
+Blower Harness
+======================
+
+.. csv-table:: **BOM**
+   :widths: 5, 60, 5, 5, 5, 10, 10, 10
+   :header-rows: 1
+   :delim: 0009
+   :file: ./blower.bom.tsv
+
+.. image:: blower.png
+  :width: 800
+  :alt: Wiring diagram
+
+You will need all the tools listed.
+
+* Using items from above BOM as well as:
+   * 3mm heat shrink tubing [:ref:`ew-shrink3`]
+   * 5mm heat shrink tubing [:ref:`ew-shrink5`]
+   * Heat gun
+   * Molex Micro-Fit 3.0 Crimper
+   * Wire Strippers
+   * Soldering iron
+   * Solder
+* Instructions:
+   #. Cut 3 pieces of the 5mm heat shrink, 15mm long each.
+   #. Thread the 5mm heat shrink onto the the entire bundle of 4 wires on the blower.  Do not shrink the heat shrink yet.
+   #. Insert the 2x pre-crimped Mega-Fit wires into the Megafit 2x2 housing.
+   #. Do a gentle pull test to confirm proper seating of the contacts.
+   #. Cut the cable at a length of 50mm from the connector.
+   #. Strip 5mm of insulation from the end of the pre-crimped wires and from the ends of the red and black pigtails of the blower.
+   #. Cut 2 pieces of 3mm heat shrink, 15mm long.
+   #. Slide the 2 pieces of heat shrink onto the red and black wires of the blower.
```

Comment:
I would maybe phrase it like "insert each piece of shrink tubing onto each one of the red and black wires" or something like that? It seemed a bit ambiguous and one might insert both on the both wires together.

Reply:
done.